lib/validations.js

```diff
@@ -88,6 +89,17 @@ validations.isAlphanumeric = function isAlphanumeric(paramName, customMessage) {
   );
 }
 
+validations.isArray = function isPlainObject(paramName, customMessage) {
```

Comment:
it looks like there's a name mismatch here. I'm assuming this function should be called `isArray`